nixos/tests/common/acme/server/default.nix

```diff
@@ -1,27 +1,27 @@
 # The certificate for the ACME service is exported as:
 #
-#   config.test-support.letsencrypt.caCert
+#   config.test-support.acme.caCert
```

Comment:
This is not true anymore since `pebble`. `pebble` generates its own CA on startup; and we download that from their internal endpoint.  This testing code is a bit messy becuase of this.

Reply:
_edit_: nvm I see we configure pebble to use this cert

Reply:
My understanding is that this is the CA that issues the static TLS certificate that is used to authenticate access to `https://acme.test/` itself, rather than the CA that Pebble uses to issue ACME certificates. Hope this makes sense... the distinction confused me at first too.

Reply:
See https://github.com/letsencrypt/pebble#avoiding-client-https-errors; we just generate our own CA rather than using the bundled one. That might not be worth it, though.

Reply:
Okay, we can't use the upstream certificates because they're only valid for `pebble` and `localhost`. We could reuse their CA since they include the private key and just issue our own certificates for the API, but at that point the simplification is marginal and I'm going to say it's out of scope for this PR.